frontend/.env.example

```
@@ -0,0 +1,11 @@
+﻿# N.E.K.O 前端环境变量配置示例
```

dotenv-linter (4.0.0)

[warning] [IncorrectDelimiter] The ﻿# N.E.K.O 前端环境变量配置示例 key has incorrect delimiter

(IncorrectDelimiter)